config/sidekiq_schedule.yml

```diff
@@ -3,6 +3,11 @@ schedule_automated_backups:
   class: "ScheduleAutomatedBackupsJob"
   queue: default
   active_job: true
+schedule_server_load_readings:
+  cron: "*/15 * * * *"
+  class: "ScheduleServerLoadReadingsJob"
+  queue: default
```

Comment:
What do you think about creating a separate queue for this? That way we can prioritise this differently from the "normal" jobs